tlslite/handshakesettings.py

```diff
@@ -231,6 +231,7 @@ def _init_misc_extensions(self):
         self.ticketKeys = []
         self.ticketCipher = "aes256gcm"
         self.ticketLifetime = 24 * 60 * 60
+        self.ticket_count = 2
```

Comment:
That's fine for now, but it would be nice for this to be settable from cmd so that future changes in tlslite wouldn't affect a test based on it.

Reply:
that's the plan